authorizer-app-backend/src/main/java/org/radarbase/authorizer/resources/SourceClientResource.kt

```diff
@@ -111,7 +111,6 @@ class SourceClientResource(
         body.deregistrations.forEach { deregistration ->
             val user = userRepository.findByExternalId(deregistration.userId, sourceType)
             if (user != null && user.accessToken == deregistration.userAccessToken) {
```

Comment:
I don't think we can do this. This will also remove this check for FitBit and other applications.
If this is a particular issue for Garmin, we should find a way to validate if the request comes from garmin and skip this check optionally.